packages/web-components/src/calendar/date-formatter.ts

```diff
@@ -0,0 +1,35 @@
+import { DateFormatter, MonthFormat, YearFormat } from '@microsoft/fast-foundation';
+
+/**
+ * Date formatting utility
+ * @public
+ */
+export class FluentDateFormatter extends DateFormatter {
+  /**
+   *
+   * @param locale - The locale data used for formatting
+   * @returns - An array of the month labels
+   * @public
+   */
+  public getMonths(locale: string = this.locale): string[] {
+    const months = Array(12)
```

Comment:
This probably won't change any time soon (😄) but we want to avoid using "magic" numbers here. Let's use a constant instead: `const NUM_MONTHS_IN_YEAR = 12;`